react-native/react/constants/platform.native.desktop.js

```diff
@@ -1,7 +1,39 @@
 'use strict'
 
 import {OS_DESKTOP} from './platform.shared'
+import path from 'path'
 
 export const isDev = process.env.NODE_ENV === 'development'
 export const OS = OS_DESKTOP
 export const isMobile = false
+
+const runMode = process.env.KEYBASE_RUN_MODE || 'devel'
+const envedPathOSX = {
+  staging: 'KeybaseStaging',
+  devel: 'KeybaseDevel',
+  prod: 'Keybase'
+}
+
+function findSocketRoot () {
+  const paths = {
+    'darwin': `${process.env.HOME}/Library/Caches/${envedPathOSX[runMode]}/`,
+    'linux': `${process.env.XDG_RUNTIME_DIR}/keybase.${runMode}/`
```

Comment:
Hm, even after the core change, this won't work, because the core change won't affect where $XDG_RUNTIME_DIR points to, and won't make $XDG_RUNTIME_DIR non-ephemeral.  What's the plan for this code?